src/editor/editor.js

```diff
@@ -89,6 +89,14 @@ export default class Editor {
 		 */
 		this.data = new DataController( this.document );
 
+		// Refresh the state of all editor commands as soon as data is ready.
+		// https://github.com/ckeditor/ckeditor5-core/issues/50
+		this.listenTo( this, 'dataReady', () => {
```

Comment:
We'll do so on destroy (we don't destroy commands now). So, I'd rather see these methods (`destroy()` and `refreshAll()` in the `CommandCollection`).